cli/commands/site/start.ts

```diff
@@ -112,7 +112,6 @@ export async function runCommand( siteFolder: string, skipBrowser = false ): Pro
 			const loggerError = new LoggerError( __( 'Failed to start site infrastructure' ), error );
 			logger.reportError( loggerError );
 		}
-		process.exit( 1 );
```

Comment:
We already set the exit code in `cli/logger.ts`. No need to explicitly exit the process in this way.